### PR TITLE
Standard-product acquisitions for hysds3

### PR DESCRIPTION
**Pull request by shitong01**
Standard-product acquisitions for hysds3

~1. revert changes to acquisitions naming~
2. correct command used for ipf_scraper
3. Fix encoding issue with `ipf_scraper_scihub`:
```
Failed to get IPF for acquisition-Sentinel-1A_20200316T110726.590Z_128_IW-esa_scihub. 
cannot use a string pattern on a bytes-like object. 
 Traceback (most recent call last):
  File "/home/ops/verdi/ops/scihub_acquisition_scraper/ipf_scrape/ipf_version.py", line 215, in 
    ipf = extract_scihub_ipf(met)
  File "/home/ops/verdi/ops/scihub_acquisition_scraper/ipf_scrape/ipf_version.py", line 180, in extract_scihub_ipf
    ipf = get_scihub_ipf(manifest)
  File "/home/ops/verdi/ops/scihub_acquisition_scraper/ipf_scrape/ipf_version.py", line 90, in get_scihub_ipf
    ns = get_scihub_namespaces(manifest)
  File "/home/ops/verdi/ops/scihub_acquisition_scraper/ipf_scrape/ipf_version.py", line 80, in get_scihub_namespaces
    matches = re.findall(r'\s+xmlns:?(\w*?)\s*=\s*[\'"](.*?)[\'"]', xml)
  File "/home/ops/verdi/lib/python3.7/re.py", line 223, in findall
    return _compile(pattern, flags).findall(string)
TypeError: cannot use a string pattern on a bytes-like object
```

crontabs:
```
# scihub acquisition scrapers
15 * * * * source $HOME/.bash_profile && $HOME/verdi/bin/python /home/ops/verdi/ops/scihub_acquisition_scraper/crons/acq_ingest_cron.py --tag develop-standard-product  --dataset_version v2.0 --hours 6 opensearch  > $HOME/verdi/log/scihub_acquisition_scraper_hour.log 2>&1
45 14 * * * source $HOME/.bash_profile && $HOME/verdi/bin/python /home/ops/verdi/ops/scihub_acquisition_scraper/crons/acq_ingest_cron.py --tag develop-standard-product  --dataset_version v2.0 --days 5 opensearch  > $HOME/verdi/log/scihub_acquisition_scraper_day.log 2>&1

# ipf scrapers
57 */6 * * * source $HOME/.bash_profile && $HOME/verdi/bin/python $HOME/verdi/ops/scihub_acquisition_scraper/crons/ipf_global_cron.py --tag=develop-standard-product > $HOME/verdi/log/cron-scrape_global_ipf.log 2>&1
```

logs:
[successful acquisition scraping](https://gist.github.com/shitong01/c18dbd413d4af686ee2e376b242007a2#file-01_acquisition_ingest)
[successful ipf scraping](https://gist.github.com/shitong01/c18dbd413d4af686ee2e376b242007a2#file-02_ipf_scraper)

### CODE REVIEW

**Comment by shitong01** [comment]
Do not merge yet, until I post the logs in the PRs.

File: acquisition_ingest/scrape_apihub_opensearch.py [hunk 1]
```diff
@@ -209,7 +209,8 @@ def create_acq_dataset(ds, met, root_ds_dir=".", browse=False):
     """Create acquisition dataset. Return tuple of (dataset ID, dataset dir)."""
 
     # create dataset dir
-    id = "acquisition-{}-esa_scihub".format(met["title"])
+    id = "acquisition-{}_{}_{}_{}-esa_scihub".format(met["platform"], get_timestamp_for_filename(met["sensingStart"]),
+                                                     met["track_number"], met["sensoroperationalmode"])
```
Comment (shitong01) [inline acquisition_ingest/scrape_apihub_opensearch.py]: Revert this back to: id = "acquisition-{}-esa_scihub".format(met["title"])

**Comment by shitong01** [comment]
@pymonger This is ready for merging / another review, added logs and verified working.

**Review by pymonger — APPROVED** [review]

**Comment by pymonger** [comment]
LGTM.
@torresal, @mlucas2018: can you also verify?

**Review by marjo-luc — APPROVED** [review]
No comments.

**Pull request merged**